src/luos.c

```diff
@@ -69,47 +69,60 @@ void Luos_Loop(void)
     {
         // There is a message available find the container linked to it
         container_t *container = Luos_GetContainer(oldest_vm);
-        // Is this container having a callback?
-        if (container->cont_cb != 0)
+        // check if this is a Luos Command
+        uint8_t cmd = 0;
+        uint16_t size = 0;
+        if (MsgAlloc_GetLuosTaskCmd(remaining_msg_number, &cmd) == FAIL)
```

Comment:
```suggestion
        while (MsgAlloc_GetLuosTaskCmd(remaining_msg_number, &cmd) != FAIL);
```
just to have less ligne in luos loop not really important

Reply:
I prefer to keep it as a if because it will be replaced by an assert one day...